### ISSUE TITLE
For some fonts such as Cascadia Code, it doesn't work

### ISSUE THREAD

**Issue by wrvsrx** [tolbertam/hyper-font-ligatures#10]
It doesn't work for Cascadia Code (A font that supports ligatures)

![image](https://user-images.githubusercontent.com/42770726/71316757-b18eef00-24b0-11ea-802b-20945d887a40.png)

But it works with Fira Code well

![image](https://user-images.githubusercontent.com/42770726/71316760-d4b99e80-24b0-11ea-978f-f487eaf0d45d.png)

And that is how Cascadia Code behaves on Windows Terminal

![image](https://user-images.githubusercontent.com/42770726/71316775-1cd8c100-24b1-11ea-9722-84f5cb2f7ee0.png)

**Comment by angelnext** [comment]
True, i wish it would be fixed, Cascadia Code is my favourite font ligature font
